src/extensions/scratch3_video_sensing/index.js

```diff
@@ -437,6 +333,21 @@ class Scratch3VideoSensingBlocks {
         const state = this._analyzeLocalMotion(util.target);
         return state.motionAmount > Number(args.REFERENCE);
     }
+
+    videoToggle (args) {
+        const state = Number(args.VIDEO_STATE);
```

Comment:
As a minor style point, similar to above: we tend to keep things as string values (emulating `enum` values) rather than using integers in situations like this. For example, I would replace `VIDEO_STATE_INFO` with something like `{OFF: 'off', ON: 'on', FLIPPED: 'on-flipped'}` (your choice on the exact wording). Since JS interns all strings there's no comparison cost, and this means both the code and the runtime values are self-documenting.

If there's specific reason to use numbers, I'd still prefer `if (state === VideoState.OFF)` or something like that -- at least the code is self-documenting, then, even if the runtime values aren't :)

Reply:
yeah - numbers suck for this - I like the idea of strings better too! :)